pkg/dockerhub/dockerhub.go

```diff
@@ -71,42 +71,54 @@ func (d *DockerHub) ListRepositories() ([]string, error) {
 }
 
 func (d *DockerHub) ListTags(repository string) ([]string, error) {
-	endpoint := fmt.Sprintf("%s/v2/repositories/%s/tags/?page_size=10000", authEndpoint, repository)
+	endpoint := fmt.Sprintf("%s/v2/repositories/%s/tags/", authEndpoint, repository)
 
 	type dockerHubTags struct {
+		Next    string `yaml:"next"`
```

Comment:
We should use sigs.k8s.io/yaml  for yaml marshalling. See https://github.com/giantswarm/fmt/blob/master/go/yaml_marshalling.md